### DISCUSSION TITLE
Assigning more space to FRITZ!NAS?

### DISCUSSION THREAD

**Discussion by gc-user** [Freetz-NG/freetz-ng#1055] [Allgemein]
I wonder if it would be possible via freetz-ng to assign more space to FRITZ!NAS (TFFS) (and not have to use a thumb drive to expand FRITZ!NAS space!).

The free FRITZ!NAS space on a 7530 is ridiculously low (~ 8,5 MB) compared to the free space shown for /var (~ 109 MB).

When browsing the file system of the fritzbox, e.g. via ftp, FRITZ!NAS shows up in /var/media/ftp/. At the same time, freetz-ng web-interface shows a lot of free space for /var in the `status` section. It would be nice to be able to assign like 10% of that free space to /var/media/ftp/.

Is that possible?

**Comment by fda77** [comment]
If you use openwrt the device will be repartitioned and there is no more dual-boot but much space for tools.
eg my 1200ac has 73,5MB filepartition

> a lot of free space for /var

ram...

**Reply by gc-user** [reply]
Thanks for the reply.

> If you use openwrt the device will be repartitioned and there is no more dual-boot but much space for tools. eg my 1200ac has 73,5MB filepartition

I like freetz and the fact that it keeps the original FOS. At the moment, I'm not yet sold on openwrt... :-)


> ram...

Ah, ok, makes sense.

Is there a folder / path within /var (other than FRITZ!NAS) that has write access? If there is one, is that linked to RAM or flash?
Maybe I can get the binary to put it temporary stuff there.

I included the binaries in the firmware in one trial freetz firmware - they could be executed but didn't function normally as they had no write access. If I can get them to write to a path other than its own (a config / compiling issue of the binaries, I assume), I wouldn't even need more FRITZ!NAS space nor the thumb drive I have to use now.

**Reply by fda77** [reply]
flash is r/o
